warehouse/query-core/src/test/java/datawave/query/jexl/visitors/RegexFunctionVisitorTest.java

```diff
@@ -186,6 +186,13 @@ public void testDoubleEndedWildCard2() throws ParseException {
         assertVisitorResult(query, query, indexOnlyFields);
     }
 
+    @Test
+    public void testMultiFieldDoubleEndedWildCard() throws ParseException {
```

Comment:
add a comment explaining why multi fielded filter functions with an index only field are okay given the presence of a double ended wildcards?

Reply:
This functionality has not changed, it simply wasn't covered before.